src/ThreeEditor/js/Viewport.js

```diff
@@ -3,15 +3,21 @@ import { TransformControls } from 'three/examples/jsm/controls/TransformControls
 import { SetPositionCommand } from './commands/SetPositionCommand';
 import { SetRotationCommand } from './commands/SetRotationCommand';
 import { SetScaleCommand } from './commands/SetScaleCommand';
-import { EditorControls } from './EditorControls';
 
 import { ViewportCamera } from './Viewport.Camera.js';
 import { ViewportInfo } from './Viewport.Info.js';
 
 import { UIPanel } from "./libs/ui";
 import { ViewHelper } from './Viewport.ViewHelper';
+import { EditorOrbitControls } from './EditorOrbitControls';
 
-export function Viewport(name, editor, { objects, grid, selectionBox }, cameraPosition) {
+import { GUI } from 'three/examples//jsm/libs/dat.gui.module.js';
+
+export function Viewport(
+    name, editor,
+    { objects, grid, planeHelpers, selectionBox },
+    { orthographic, cameraPosition, clipPlane, planePosLabel, planeHelperColor, showPlaneHelpers, gridRotation = new THREE.Euler(0, Math.PI / 2, 0) } = {}
```

Comment:
Why `new THREE.Euler(0, Math.PI / 2, 0)` ?

Reply:
To set the grid to default rotation, that is perpendicular to the Y-axis.

Reply:
Great, but next generation of developers would need some amount of time to figure it out.
Therefore - please add a comment, something like that:
"setting default rotation(?) for the default view, so the users would see XZ plane.
The rotation is defined by Euler angles (0, pi/2, 0)."